### ISSUE TITLE
ABS Icon smushed

### ISSUE THREAD

**Issue by raviwarrier** [advplyr/audiobookshelf-app#1105]
![Screenshot_2024-02-26-23-41-05-02_e9782661401ed6f222682e1e02b74bd4.jpg](https://github.com/advplyr/audiobookshelf-app/assets/42061703/baa7ff8e-5a18-48c6-bd18-a32c8b913e80)

Don't know if this is new or local, but just saw this.

Android 14. Latest app version (can't remember version, but as of 27th Feb 2024, whatever is the latest version)

**Comment by raviwarrier** [comment]
@nichwall yup, it's because of the download circle. forgot to mention it in my half-asleep state. :)

if its the same problem, please go ahead and merge the two issues or close this one. 

Cheers buddy.

**Comment by advplyr** [comment]
What device is that? I'm not sure how to handle this for that narrow of devices. I'm thinking of moving the search to inside the hamburger menu or the cast icon

**Comment by raviwarrier** [comment]
It's one plus nord 3e. 
What I was thinking was merge the download progress with the ABS icon. You can do this in multiple ways and I will leave that upto you - a halo ring around the icon, or filling/emptying of the image related to percentage, etc.

when downloading, tapping on the icon takes you to the downloading screen, else, if maintains it current/regular functionality.

Search is perhaps one of the most used features and UI-wise, it would better to leave it outside... If you have to move anything inside the hamburger, move the cast icon.
